source/index.html.slim

```diff
@@ -29,13 +29,14 @@
 	          a href=next_event.wiki_url wiki
 	          = " or "
 	          a href=next_event.facebook_url "comment it on facebook"
-          .upcoming_events
-            h3 Upcoming Events
-            hr
-            -upcoming_events.reverse.each_with_index do |upcoming_event|
-              h3
-                =upcoming_event.date
       .triangle
+      .event-box-upcoming
+        h3 Upcoming Events
+        ul
+          -upcoming_events.reverse.each_with_index do |upcoming_event|
```

Comment:
I'm asking, because I would expect that the events are already in the correct order.  Was just asking out of curiousity.